packages/Tabs/src/Tabs.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+
+import './tabs.scss';
+
+import { useState } from 'react';
+import classNames from 'classnames';
+import { NavLink } from 'react-router-dom';
+
+import CrownSolid from '@igloo-ui/icons/dist/CrownSolid';
+import TextBulletSolid from '@igloo-ui/icons/dist/TextBulletSolid';
+
+export interface TabInterface {
+  /** The tab pane contents of the tabs list */
+  children?: React.ReactNode;
+  /** Add an id attribute to the tab */
+  id?: string;
+  /** Add text for the tab */
+  label: string;
+  /** Add a notification indicator to the tab */
+  notification?: boolean;
+  /** Add a premium indicator to the tab */
+  premium?: boolean;
+  /** Use this to redirect to another route */
+  to?: string;
+}
+
+export interface TabsProps extends React.ComponentProps<'div'> {
+  /** The function that's called when a tab is clicked */
+  onSelect?: () => void;
+  /** The index number of the Tab that should be selected on load */
+  defaultIndex?: number;
+  /** Add a data-test tag for automated tests */
+  dataTest?: string;
+  /** Add a specific class to the Tabs component */
+  className?: string;
+  /** Add an inline style to the Tabs */
+  isInline?: boolean;
+  /** An array of objects that specify the props of the tab buttons */
+  tabs: Array<TabInterface>;
+}
+
+const Tabs: React.FunctionComponent<TabsProps> = ({
+  onSelect,
+  defaultIndex = 0,
+  dataTest,
+  className,
+  isInline = true,
+  tabs,
+}) => {
+  const classes = classNames('ids-tabs', className, {
+    'ids-tabs--inline': isInline,
+    'ids-tabs--heading': !isInline,
+  });
+
+  const [selectedTabIndex, setSelectedTabIndex] =
+    useState<number>(defaultIndex);
+  const handleOnClick = (index: number): void => {
+    if (setSelectedTabIndex) {
+      setSelectedTabIndex(index);
+    }
+    if (onSelect) {
+      onSelect();
+    }
+  };
+
+  const renderTabButton = (
+    tab: TabInterface,
+    index: number
+  ): React.ReactNode => {
+    let button = (
+      <button
+        className="ids-tabs__tab-button"
```

Comment:
As we are in the block `tab` it is better to use this nomenclature `ids-tab__btn` for CSS. 
And to keep a consistency with the other components replace **button** by **btn**

Reply:
alright, thanks for the tip